Automattic/sensei
# Add onboarding theme step backend

Part of https://github.com/Automattic/sensei/issues/6195

### Changes proposed in this Pull Request

* It adds the backend to the theme step.
  * It doesn't have the logic to install the theme yet. Currently, it's a `Promise.resolve()` that should be replaced with the fetch to the endpoint that will install and activate the theme.
* This PR also updates some copy and images. More context here: p1670017026866649-slack-C013QUH20TS

### Testing instructions


* Go to the Setup Wizard (`/wp-admin/admin.php?page=sensei_setup_wizard`).
* In the theme step, click on "Get the Course theme".
* Complete the setup wizard, and make sure that in the last step you see the option "Installing the Course theme".
* Go through the setup wizard again, but this time select "Skip theme selection", and make sure in the last step you don't see the option "Installing the Course theme".


### Screenshot / Video

https://user-images.githubusercontent.com/876340/205702416-f05d5b40-7452-489c-aeb0-c645868ad700.mov
